scripts/upgrade.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/sh
```

Comment:
Nice, maybe at the beginning of this file we can just add as comments some examples on how to run this script. For example:
- What command to run deploy? on prod? on staging?
- What command to run test? on prod? on staging?

Just for clarity and not to forget anything

Reply:
I will see if I can add a help me command